plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1147,6 +1184,60 @@ func (c *PodAffinityChecker) satisfiesExistingPodsAntiAffinity(pod *v1.Pod, meta
 			return false
 		}
 	}
+
```

Comment:
Given the new size of the function, what do you think about placing this new block of code in its own function? Something like SatisfiesMaxPodsPerToplogy.